internal/core/regularsurface.cpp

```diff
@@ -95,24 +95,30 @@ Point RegularSurface::from_cdp(
     return this->m_plane.m_inverse_transformation * point;
 }
 
-float RegularSurface::value(
-    std::size_t const row,
-    std::size_t const col
-) const noexcept (false) {
-    if (row >= this->nrows()) throw std::runtime_error("Row out of range");
-    if (col >= this->ncols()) throw std::runtime_error("Col out of range");
 
-    return this->m_data[row * this->ncols() + col];
-};
+std::pair<std::size_t,std::size_t> RegularSurface::to_pair(std::size_t row, std::size_t col) {
```

Comment:
It is convenient to have code that is only used in one class as static function in that class. Obesely, it could be anywhere. Thus, I moved out to simplify the notation. Any opinion on the new variant?